benchmark/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ add_rocprim_benchmark_hip(benchmark_hip_warp_sort.cpp)
 add_rocprim_benchmark_hip(benchmark_hip_device_memory.cpp)
 
 # rocPRIM HC benchmarks
+if(HIP_COMPILER STREQUAL "HCC")
```

Comment:
I think you should also disable HC example in `https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/master/example/CMakeLists.txt`

Reply:
Right, I had examples OFF during my compiles, I'll add this